hawkular-datamining-forecast/src/test/java/org/hawkular/datamining/forecast/AugmentedDickeyFullerTestTest.java

```diff
@@ -42,14 +42,39 @@ public void testAustouristsInterceptTimeTrend() throws Exception {
         double statistics = augmentedDickeyFullerTest.statistics();
         double pValue = augmentedDickeyFullerTest.pValue();
 
-        System.out.println(AugmentedDickeyFullerTest.Type.InterceptTimeTrend.name() + "=" + statistics + ", pValue="
-                + pValue);
-
         Assert.assertEquals(-7.4083, statistics, 0.001);
         Assert.assertTrue(pValue <= 0.01);
     }
 
     @Test
+    public void testAustourists4Lags() throws IOException {
+        ModelData rModel = ModelReader.read("austourists");
+        double[] x = Utils.toArray(rModel.getData());
+
+        AugmentedDickeyFullerTest augmentedDickeyFullerTest = new AugmentedDickeyFullerTest(x, 4,
+                AugmentedDickeyFullerTest.Type.InterceptTimeTrend);
+        double statistics = augmentedDickeyFullerTest.statistics();
+        double pValue = augmentedDickeyFullerTest.pValue();
+
+        Assert.assertEquals(-1.8381, statistics, 0.001);
+        Assert.assertEquals(0.6388, pValue, 0.05);
+    }
+
+    @Test
+    public void testAustourists4LagsInterceptNoTimeTrend() throws IOException {
+        ModelData rModel = ModelReader.read("austourists");
+        double[] x = Utils.toArray(rModel.getData());
+
+        AugmentedDickeyFullerTest augmentedDickeyFullerTest = new AugmentedDickeyFullerTest(x, 4,
+                AugmentedDickeyFullerTest.Type.InterceptNoTimeTrend);
+        double statistics = augmentedDickeyFullerTest.statistics();
+        double pValue = augmentedDickeyFullerTest.pValue();
+
+        Assert.assertEquals(-0.0625, statistics, 0.001);
+        Assert.assertEquals(0.9451, pValue, 0.02);
```

Comment:
?, the first param in `Assert.assertEquals` should be the message, shouldn't it? Perhaps, use more comments.